x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/rule_details.tsx

```diff
@@ -521,30 +406,28 @@ export const RuleDetails: React.FunctionComponent<RuleDetailsProps> = ({
                 color="warning"
                 data-test-subj="actionWithBrokenConnectorWarningBanner"
                 size="s"
-                title={i18n.translate(
-                  'xpack.triggersActionsUI.sections.ruleDetails.actionWithBrokenConnectorWarningBannerTitle',
-                  {
-                    defaultMessage:
-                      'There is an issue with one of the connectors associated with this rule.',
-                  }
-                )}
               >
-                {hasEditButton && (
-                  <EuiFlexGroup gutterSize="s" wrap={true}>
-                    <EuiFlexItem grow={false}>
-                      <EuiButton
-                        data-test-subj="actionWithBrokenConnectorWarningBannerEdit"
-                        color="warning"
-                        onClick={() => setEditFlyoutVisibility(true)}
-                      >
-                        <FormattedMessage
-                          id="xpack.triggersActionsUI.sections.ruleDetails.actionWithBrokenConnectorWarningBannerEditText"
-                          defaultMessage="Edit rule"
-                        />
-                      </EuiButton>
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                )}
+                <p>
+                  <EuiIcon color="warning" type="alert" />
+                  &nbsp;
+                  <FormattedMessage
+                    id="xpack.triggersActionsUI.sections.ruleDetails.actionWithBrokenConnectorWarningBannerTitle"
+                    defaultMessage="There is an issue with one of the connectors associated with this rule."
```

Comment:
<img width="1594" alt="Screen Shot 2022-04-01 at 11 15 21 AM" src="https://user-images.githubusercontent.com/13104637/161292244-f47b1c4e-63c2-41cc-ad90-53e6f9da45ec.png">

Something wrong with the icon when there is a broken action attached the rule.